lib/src/easy_localization_app.dart

```diff
@@ -43,11 +43,15 @@ class EasyLocalization extends StatefulWidget {
 class _EasyLocalizationLocale extends ChangeNotifier {
   Locale _locale;
   static Locale _savedLocale;
+  // Get default OS Locale
+  static final _osCurrentLocale = Intl.getCurrentLocale().split("_");
 
   // @TOGO maybe add assertion to ensure that ensureInitialized has been called and that
   // _savedLocale is set.
   _EasyLocalizationLocale(Locale fallbackLocale)
-      : this._locale = _savedLocale ?? fallbackLocale;
+      // if _savedLocale and fallbackLocale null init by default OS Locale
+      : this._locale = (_savedLocale ?? fallbackLocale) ??
```

Comment:
If the device local is not in supportedLocals and language files do not exist for this locale, what will happen? 
I think localization will fail no?

Reply:
we can resolve by taking from `supportedLocales`

```dart
// if _savedLocale and fallbackLocale null and default OS Locale in supportedLocales 
  // init by default OS Locale else init by supportedLocales[0]
      : this._locale = (_savedLocale ?? fallbackLocale) ??
      supportedLocales.firstWhere((local) => local ==_osLocal, orElse: () => supportedLocales.first); 
```